libs/application/templates/p-sign/src/lib/pSignTemplate.ts

```diff
@@ -22,6 +23,7 @@ const PSignTemplate: ApplicationTemplate<
   name: 'Stæðiskort',
   dataSchema: dataSchema,
   readyForProduction: true,
+  allowedDelegations: [AuthDelegationType.Custom],
```

Comment:
For the purpose of deploy feature testing I have this as custom to be able to test with gervimaður Afríka as subject and gervimaður Útlönd as actor. Gervimaður Afríka is the only gervimaður with læknisvottorð for the pSign application checks.